src/components/SensorMap/SensorMap.js

```diff
@@ -13,9 +13,19 @@ const GET_SENSORS = gql`
       loc_y
       temperature
       humidity
+    },
+    sensorfaults {
```

Comment:
This will not work with the latest version of the backend 
```suggestion
    SensorFaults {
```